### DISCUSSION TITLE
How to have a preview image for model ?

### DISCUSSION THREAD

**Discussion by tuannguyensn2001** [google/model-viewer#3058] [Q&A]
### Description

I have a task to take preview image for each model.  I try to capture screen by using html2canvas but it didn't work, i see on the docs it only has export gltf, glb

#### Live Demo
https://modelviewer.dev/
I want to capture the image in model

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
I believe you're looking for  this  function: https://modelviewer.dev/docs/index.html#entrydocs-loading-methods-toBlob

**Reply by tuannguyensn2001** [reply]
thank you so much, it works for me
